polled_camera/README.md

```diff
@@ -0,0 +1,41 @@
+## `polled_camera` Overview
+
+`polled_camera` defines the ROS interface that client nodes use to request images from a polling camera driver node (e.g. [prosilica_camera](https://index.ros.org/p/prosilica_camera/)). The protocol is:
+
+* The camera driver advertises a service call `<camera>/request_image` of type [polled_camera/GetPolledImage](http://www.ros.org/doc/api/polled_camera/html/srv/GetPolledImage.html).
```

Comment:
link is for ROS 1, maybe use this one https://github.com/ros-perception/image_common/blob/rolling/polled_camera/srv/GetPolledImage.srv ?

Reply:
```suggestion
* The camera driver advertises a service call `<camera>/request_image` of type [polled_camera/GetPolledImage](https://github.com/ros-perception/image_common/blob/rolling/polled_camera/srv/GetPolledImage.srv).
```